File: src/main/Department.java
package main;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.ArrayList;
import java.util.List;

public class Department {

    @JsonProperty("id")
    private String id;

    @JsonProperty("departmentName")
    private String departmentName;

    @JsonProperty
    private List<String> courseSectionIds = new ArrayList<>();

    public Department() {

    }

    public List<String> getCourseSectionIds() {
        return courseSectionIds;
    }

    public void setCourseSectionIds(List<String> courseSectionIds) {
        this.courseSectionIds = courseSectionIds;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getDepartmentName() {
        return departmentName;
    }

    public void setDepartmentName(String departmentName) {
        this.departmentName = departmentName;
    }

}

File: main/Department.py
from typing import List

from main.CourseSection import CourseSection


class Department:
    __id : str
    __departmentName : str
    __courseSectionIds : List[str]

    def __init__(self, id : str, departmentName : str, courseSectionIds : List[str]):
        self.__id = id
        self.__departmentName = departmentName
        self.__courseSectionIds = courseSectionIds

    def getId(self):
        return self.__id

    def setId(self, id : str):
        self.__id = id

    def getDepartmentName(self):
        return self.__departmentName

    def setDepartmentName(self, departmentName : str):
        self.__departmentName = departmentName

    def getCourseSectionIds(self):
        return self.__courseSectionIds

    def setCourseSectionIdS(self, courseSectionIds : List[str]):
        self.__courseSectionIds = courseSectionIds